src/pages/ProductDetail/ProductDetail.js

```diff
@@ -59,10 +154,24 @@ const ProductDetail = () => {
             className="btn cart-btn"
             onClick={() => {
               setIsModalOpen(true);
+              shoppingBasket();
             }}
           >
             장바구니 담기
           </button>
+          <button
+            className="btn wishlist-btn"
+            onClick={() => {
+              handleWishBtn();
+            }}
```

Comment:
```suggestion
            onClick={handleWishBtn}
```
넘겨주는 인자가 없다면 함수이름으로 그대로 연결해주세요~